#!/usr/bin/env python
#PACKAGE = "two_scara_collaboration"

#from dynamic_reconfigure.parameter_generator_catkin import *

#gen = ParameterGenerator()
#gen.add("wrench_force_y",    double,    0, "force at object along direct of conveyor", 6,  4, 9)

#exit(gen.generate(PACKAGE, "cylinder_blocks_poses_publisher", "DynamicParams"))